vignettes/data-transform-as-shiny-module.Rmd

```diff
@@ -12,13 +12,13 @@ vignette: >
 
 ## Introduction
 
-`teal` version `0.16` introduced new argument in `teal::module` called `transformers`. 
+`teal` version `0.16` introduced new argument in `teal::module` called `transformators`. 
 This argument allows to pass a `list` of `teal_data_module` objects that are created using `teal_transform_module()` function.
 
 The main benefit of `teal_transform_module()` is the ability to transform data before passing it 
```

Comment:
There is a statement in line 21: "teal_transform_module()` is a function ...". Move it here - upgfront. Let's introduce the function first and then discuss it's benefits.

Reply:
29ea4f17cc7008c4b9731fd6889b2fd8847cc067